src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -430,12 +453,15 @@ def __init__(self, config):
         self.dropout = nn.Dropout(config.hidden_dropout_prob)
 
     def forward(self, hidden_states, input_tensor):
+        self.io = hidden_states # TODO:  remove this
         hidden_states = self.dense(hidden_states)
+        self.o = hidden_states # TODO:  remove this
```

Comment:
Difference in input of `dense` layer in `BigBirdOutput` is `0.0004749298` (when comparing with orig. BigBird) while the difference in output of `dense` layer in `BigBirdOutput` is `0.011826634`.
So, looks like there is some problem in dense layer. But I am unable to fix it. Please help me in this.

Reply:
It might be that the error already happened before. One thing I can think of is that the `"gelu"` functions are actually not exactly the same. In `transfomers` we have three different `gelu` functions: `gelu_new`, `gelu_fast` and `gelu`. I think in the case of BigBird it's actually `gelu_fast` and not `gelu` . Let me know if this works :-)

Reply:
Thanks! it's working when replacing `gelu` with `gelu_fast`.